lib/src/presentation/widget/grid_card_item.dart

```diff
@@ -34,6 +34,9 @@ class GridCardItem extends StatelessWidget {
           if (image != null)
```

Comment:
Instead to and if-else without {}, you can use a ternary operator with ? and :

Reply:
Done.